Reorganize consistency check sections (#37)
ietf-wg-ohai/draft-ohai-svcb-config: draft-ietf-ohai-svcb-config.md

on the well-known gateway location {{gateway-location}}.
Use of encrypted DNS along with DNSSEC can also be used as a mitigation.

When clients fetch a gateway's configuration ({{config-fetch}}),
they can expose their identity in the form of an IP address if they do not
connect via a proxy or some other IP-hiding mechanism. In some circumstances,
this might not be a privacy concern, since revealing that a particular
client IP address is preparing to use an Oblivious HTTP service can be

> expected. However, if a client is otherwise trying to obfuscate its IP

Reviewer: ```suggestion
expected. However, if a client is otherwise trying to hide its IP
```

Reviewer: ... for consistency with "IP-hiding" above.

address or location (and not merely decouple its specific requests from its

> IP address), or if revealing its IP address facilitates key targeting attacks 

Author: ```suggestion
IP address), or if revealing its IP address facilitates key targeting attacks
```

> IP address), or revealing its IP address will increase the risk of a key
> targeting attack (if a gateway service is trying to differentiate traffic
> across client IP addresses), a proxy or similar mechanism can be used to fetch

Reviewer: ```suggestion
IP address), or if revealing its IP address facilitates key targeting attacks 
(if a gateway service uses IP addresses to associate specific configurations
with specific clients), a proxy or similar mechanism can be used to fetch
```

attacks. {{?CONSISTENCY=I-D.ietf-privacypass-key-consistency}} provides an analysis
of the options for ensuring the key configurations are consistent between
different clients. Clients SHOULD employ some technique to mitigate key
targeting attacks, such as the option of confirming the key with a shared
proxy as described in {{CONSISTENCY}}. Oblivious gateways that are detected
to use targeted key configurations per-client MUST NOT be used.

## dohpath Targeting Attacks

For oblivious DoH servers, an attacker could use unique `dohpath` values
to target or identify specific clients. This attack is very similar to

> the key targeting attack described above.

Reviewer: ```suggestion
the generic OHTTP key targeting attack described above.
```

Reviewer: ... to make it more clear why there's a separate section for the dohpath.

> Clients SHOULD mitigate such potential attacks. This can be done with

Reviewer: ```suggestion
Clients SHOULD mitigate such attacks. This can be done with a
```
